WordPress/gutenberg
# Fix : Snackbar Notice Inconsistency

## What?
Fixes: https://github.com/WordPress/gutenberg/issues/66364


- Enable `View Options` other than `Distraction Fee` to show a `snackbar notice` when they are `toggled`
- `View Options` affected : `Top toolbar`  `Spotlight mode`  `Fullscreen mode`

## Why?

- https://github.com/WordPress/gutenberg/issues/66364
> Either all these options should trigger a snackbar notice or none.
I'd like to propose to remove the snackbar notice of the Distraction free mode for consistency and simplicity. I'm not sure this snackbar notice adds much value.

> Note: recommended to check what happens when toggling these options via the Command palette.

## How?

- Create toggle actions similar to `toggleDistractionFree` for other `View Options`
- Replace hardcoded functionality with these `toggles` which allows `Snackbar Notice` to be visible when these options are toggled via the `More Menu`

## Testing Instructions




1. Edit a post.
2. Go to the top bar > Options.
3. Toggle on and off each view option.
4. Observe all modes triggers a snackbar notice.
